snuba/datasets/spans_processor.py

```diff
@@ -90,7 +91,9 @@ def __safe_extract_int(
             metrics.increment("missing_field", tags={"field": field})
         return None
 
-    def process_message(self, message, metadata) -> Optional[ProcessedMessage]:
+    def process_message(
+        self, message: Sequence[Any], metadata: KafkaMessageMetadata
+    ) -> Optional[ProcessedMessage]:
         if not (isinstance(message, (list, tuple)) and len(message) >= 2):
```

Comment:
I'm not sure when this would be a list vs tuple, so I just kept the type as  `Sequence[Any]` to cover both scenarios